PSTenable/Public/Get-PSTenableSeverity.ps1

```diff
@@ -26,66 +26,93 @@ function Get-PSTenableSeverity {
             'Medium',
             'Low'
         )]
-        [string]
-        $Severity
+        [string[]]

```

Comment:
needed so the validater will accepts multiple values, but not any value not in the list